code/modules/core_implant/cruciform/machinery/cloning.dm

```diff
@@ -400,6 +400,9 @@
 		var/obj/item/stack/material/biomatter/B = I
 		if (B.biomatter_in_sheet && B.amount)
 			var/sheets_amount_to_transphere = input(user, "How many sheets you want to load?", "Biomatter melting", 1) as num
+			if (sheets_amount_to_transphere > B.amount)
```

Comment:
Why not just make it set it to the amount that you have so that you can enter 99 when you have 20 sheets and it puts in your 20 sheets vs making you redo the interaction to get the exact number right

Reply:
sure can do.